base/util.jl

```diff
@@ -445,9 +445,21 @@ function securezero! end
     ccall(:memset, Ptr{T}, (Ptr{T}, Cint, Csize_t), p, 0, len*sizeof(T))
 unsafe_securezero!(p::Ptr{Cvoid}, len::Integer=1) = Ptr{Cvoid}(unsafe_securezero!(Ptr{UInt8}(p), len))
 
+"""
+    Base.getpass(message::AbstractString) -> `Base.SecretBuffer`
+
+Display a message and wait for the user to input a secret, returning an `IO`
+object containing the secret.
+
+Note that on Windows, the secret might be displayed as it is typed; see
+`Base.winprompt` for securely retrieving username/password pairs from a secure
```

Comment:
`winprompt` seems misnamed now… should it be `wingetpass`?

Reply:
(also, the word "secure" appears in this sentence twice, which seems a bit redundant)

Reply:
It returns a username/password pair.  It's more like `wingetcredentials` or some such.  But these are all un-exported functions and can be changed down the line.  I'm inclined to leave it be.